### DISCUSSION TITLE
Welcome to AHRS!

### DISCUSSION THREAD

**Discussion by Mayitzin** [Mayitzin/ahrs#13] [General]
## Welcome!
  AHRS is now using Discussions to connect with members using the package. Here we all can:
  * Ask questions about its contents.
  * Ask for features.
  * Discuss ideas to improve.
  * Show some nice implementations and uses.

Simply comment below with an introduction of yourself and share what's in your mind about this community.

Be respectful and clear, so that everyone can take part and create a plethora of minds playing around.

**Comment by moutayam** [comment]
That's awesome!!
Keep up the good work.
